types/detective/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@types/acorn": "*"
```

Comment:
```suggestion
    "@types/acorn": "4.0.6"
```

Reply:
^7.0.0 was the latest version acorn version officially tested within native package
Yes, we can switch to native acorn on DT but for `detective` we should probably use same version as native package